src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs

```diff
@@ -248,6 +252,57 @@ public ProcessResult SendSupplierAssemblyEmail(int orderNumber)
             return new ProcessResult(true, $"Email sent for purchase order {orderNumber} to Logistics");
         }
 
+        public ProcessResult SendFinanceAuthRequestEmail(int currentUserId, int orderNumber)
+        {
+            var order = this.GetOrder(orderNumber);
+
+            var user = this.employeeRepository.FindById(currentUserId);
+            var orderUrl = $"{this.appRoot}/purchasing/purchase-orders/{orderNumber}";
+            var emailBody = $"Purchasing have raised order {orderNumber} for {order.Supplier.Name}.\n"
+                            + $"{user.FullName} would like you to Authorise it which you can do here:\n"
+                            + $"{orderUrl} \n"
+                             + $"Thanks";
+
+            var email1 = this.employeeRepository.FindById(32864).PhoneListEntry.EmailAddress;
+            var email2 = this.employeeRepository.FindById(32835).PhoneListEntry.EmailAddress;
+            var email2Name = this.employeeRepository.FindById(32835).FullName;
+            var cc = new List<Dictionary<string, string>>
+                              {
+                                  new Dictionary<string, string>
+                                      {
+                                          { "name", email2Name },
+                                          { "address", email2 }
+                                      }
+                              };
+            var bcc = new List<Dictionary<string, string>>
```

Comment:
Does bcc definitely work? I have a vague memory of it not working for some reason

Reply:
Yeah pretty sure it does, I used it for the PO pdf emails if folk change the slider to bcc themselves!